### ISSUE TITLE
Tuist support

### ISSUE THREAD

**Issue by st-small** [TimOliver/TOCropViewController#548]
Hi folks! Does anybody use CropViewController in the project with Tuist? When I try to set dependency I got an error that CropViewController is missing. Is it possible to fix them?

**Comment by nickaroot** [comment]
Tuist glitch, Swift Package Manager headers not added automatically (sometimes)

**Fix (Tuist 4.6.0):**

Add _HEADER_SEARCH_PATHS_, _OTHER_CFLAGS_ and _OTHER_SWIFT_FLAGS_  to any target importing TOCropViewController like this:


```swift
let project = Project(
    ...
    settings: .settings(
        base: [
            "HEADER_SEARCH_PATHS": [
                "$(SRCROOT)/Tuist/.build/checkouts/TOCropViewController/Objective-C/TOCropViewController/include/TOCropViewController",
            ],
            "OTHER_CFLAGS": [
                "-fmodule-map-file=$(SRCROOT)/Tuist/.build/tuist-derived/TOCropViewController/TOCropViewController.modulemap",
            ],
            "OTHER_SWIFT_FLAGS": [
                "-Xcc",
                "-fmodule-map-file=$(SRCROOT)/Tuist/.build/tuist-derived/TOCropViewController/TOCropViewController.modulemap",
            ],
        ],
        ...
    )
    ...
)
```

**Comment by TimOliver** [comment]
Thanks so much for resolving that! I would have had no idea what was going on. 😅
